opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -586,7 +586,7 @@ namespace Opm {
 
             try {
                 well->wellTesting(ebosSimulator_, simulationTime, this->wellState(), this->groupState(), wellTestState(), deferred_logger);
-            } catch (std::runtime_error& e) {
+            } catch (std::exception& e) {
```

Comment:
Sorry for the delayed response.  This should probably be a reference to `const`, i.e.,
```C++
catch (const std::exception& e)
````
instead.